miniapps/src/main/java/mx/yellowme/youst/miniapps/utils/HapticFeedbackHelper.kt

```diff
@@ -0,0 +1,28 @@
+package mx.yellowme.youst.miniapps.utils
+
+import android.content.Context
+import android.os.Build
+import android.os.VibrationEffect
+import android.os.Vibrator
+
+object HapticFeedbackHelper {
+
+    private const val NEWER_DEVICES_MS = 3L
+    private const val OLDER_DEVICES_MS = 15L
+
+    fun generateTouchFeedback(context: Context) {
```

Comment:
Maybe we could rename this to `generateTouchFeedbackOn`, so calling the method might look like this:

```
HapticFeedbackHelper.generateTouchFeedbackOn(this)
```